ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/domain/CCDPaymentOption.java

```diff
@@ -2,7 +2,7 @@
 
 public enum CCDPaymentOption {
     IMMEDIATELY("immediately"),
-    FULL_BY_SPECIFIED_DATE("on or before specified date"),
+    BY_SPECIFIED_DATE("on or before specified date"),
```

Comment:
I suppose you need to update this as well:

src/integration-test/tests/citizen/ccj/pages/payment-options.ts:5

```
const fields = {
  paymentOption: {
    Immediate: 'input[id=optionIMMEDIATELY]',
    Instalments: 'input[id=optionINSTALMENTS]',
    FullBySetDate: 'input[id=optionFULL_BY_SPECIFIED_DATE]'
  }
}
```

Reply:
there is another PR for it mate

Reply:
why? It's the same git repo